.gitattributes

```diff
@@ -1 +1,2 @@
-*.sh text eol=lf
\ No newline at end of file
+*.sh text eol=lf
+gradlew.bat text eol=crlf
```

Comment:
Have you tried to re-do the command ( `./gradlew wrapper --gradle-version=4.7` ) after adding this line? I still wonder why the bat file's change is so big.

@kiview I remember you had issues with that bat file on Windows, could you please verify that it is working in this branch?

Reply:
Yes, I did it again. The reason is, that git didn't change in the first place. The current file from the master has the "wrong" line-ending contained. This commit just "fixes" that.

Reply:
👍 I'll summon @kiview to test it since he has a Windows environment, but looks ok in general :)

Reply:
Alright, I'll try tonight :wink:

Reply:
I think something is slightly off here. 
After checking out the branch, I directly have a changed `gradlew.bat` in my directory, having CRLF line endings.
Even after doing a `git checkout gradlew.bat` it is directly changed again. `git diff` shows a `^M` after every line (so CRLF added).

I have no global autcrlf config in my Git.

It might help if I commit this file to the PR?

Reply:
You know what. I removed all the `crlf` stuff and just committed the actual change. Actually this seems to be sufficient.